app/stacks/global/front-door/waf-policy.tf

```diff
@@ -26,6 +26,23 @@ resource "azurerm_frontdoor_firewall_policy" "default" {
       }
     }
 
+    override {
+      rule_group_name = "LFI"
+
+      rule {
+        # Possible Remote File Inclusion (RFI) Attack: Off-Domain Reference/Link

```

Comment:
Can we change this comment to the rule descriptor?: `Path Traversal Attack`
https://learn.microsoft.com/en-us/azure/web-application-firewall/afds/waf-front-door-drs?tabs=drs20#drs930-20